api/db-cdr/generate-cdr/create-tables.sh

```diff
@@ -22,62 +22,61 @@ for filename in generate-cdr/bq-schemas/*.json;
 do
     json_name=${filename##*/}
     table_name=${json_name%.json}
-    if [[ "$table_name" != 'ds_procedure_occurrence_52' || "$table_name" != 'prep_survey']]
+    # do not delete existing tables
+    if [[ ! "$table_name" =~ ds_procedure_occurrence_52|prep_survey|prep_survey_concept_ancestor ]]
```

Comment:
We can remove api/db-cdr/generate-cdr/ds_procedure_occurrence_52.json and api/db-cdr/generate-cdr/prep_survey_concept_ancestor.json.  I don't see anything in the code that references either.